modules/local/multiqc.nf

```diff
@@ -1,70 +1,69 @@
 process MULTIQC {
     label 'process_medium'
-
-    conda "bioconda::multiqc=1.23"
-    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/multiqc:1.23--pyhdfd78af_0' :
-        'biocontainers/multiqc:1.23--pyhdfd78af_0' }"
+    conda "bioconda::multiqc=1.25.1"
```

Comment:
These changes have already been included on #436, if would be great if you could do a clean PR only with the fix